docs/README.de.md

```diff
@@ -0,0 +1,91 @@
+<p align="center"><img src="/src/icons/icon@128.png" alt="holo-schedule logo"></p>
+
+# holo-schedule
+
+![Test status](https://github.com/YunzheZJU/holo-schedule/workflows/Test/badge.svg)
+
+—— Keep in touch with Hololive.
+
+Eine browser extension für alle Livestreams
+
+[中文](/docs/README.zh_CN.md) [日本語](/docs/README.ja.md) [Eng](/README.md)
+
+### Inhaltsverzeichnis
+
+Du kannst die Erweiterung auf den offiziellen Stores downloaden
+[Mozilla AMO](https://addons.mozilla.org/firefox/addon/holo-schedule/) 
+oder
+[Chrome Web Store](https://chrome.google.com/webstore/detail/holoschedule/fjicegllhddldnnkgfefblholeegpcad).
+
+[![Mozilla AMO](/docs/get-the-add-on.png)](https://addons.mozilla.org/firefox/addon/holo-schedule/)
+[![Chrome Web Store](/docs/available-in-the-chrome-web-store.png)](https://chrome.google.com/webstore/detail/holoschedule/fjicegllhddldnnkgfefblholeegpcad)
+
+## Unterstützte Browser
+
+| Browser                   | Support Level                                                                                      |
+| ------------------------- | -------------------------------------------------------------------------------------------------- |
+| Firefox (>=57.0)          | *Officially Supported* v57 is the minimum version required to support Firefox's modern add-on.     |
+| Chrome (>=57.0)           | *Officially Supported* v57 is the minimum version required to support CSS Grid layout.             |
+| Opera / Edge (>=79.0.309) | *Unofficially Supported* as a Chrome-compatible target.                                            |
```

Comment:
I found some sentences left untranslated. It's totally OK if this is intended.

Reply:
Ok gotchu' i translated them in a new update, will let you know. Thanks for the reminder